src/dataExplorer/components/ResultsPane.tsx

```diff
@@ -128,25 +124,14 @@ const ResultsPane: FC = () => {
 
       switch (language) {
         case LanguageType.SQL:
-          if (isFlagEnabled('v2privateQueryUI')) {
-            url = `${API_BASE_PATH}api/v2private/query?database=${selection.bucket?.name}`
-            // the `\n` character is interpreted by the browser as a line break
-            // so replacing the `\n` char with empty space to keep query in correct syntax
-            inputValue = text.replace(/\n/g, ' ')
-            // Intentionally keep the `break` inside the if statement;
-            // Otherwide, users cannot download csv for SQL queries when
-            // the feature flag is not enabled
-            break
-          }
-        // If the flag is not enabled, each SQL query actually will be wrapped
-        // inside a Flux query `iox.sql()`, and the if statement will be false,
-        // so we want the logic to fall to the next case LanguageType.FLUX
+          url = `${API_BASE_PATH}api/v2private/query?database=${selection.bucket?.name}`
+          // the `\n` character is interpreted by the browser as a line break
+          // so replacing the `\n` char with empty space to keep query in correct syntax
+          inputValue = text.replace(/\n/g, ' ')
+          break
         case LanguageType.FLUX:
           url = `${API_BASE_PATH}api/v2/query?${new URLSearchParams({orgID})}`
-          const query =
-            language == LanguageType.SQL
```

Comment:
Will this cause a different behavior when the flag is removed, since we are no longer checking for whether language is `LanguageType.SQL`? Or was `language == LanguageType.SQL` only when the feature flag was present?

Reply:
Good question! Since we are removing the feature flag, so `language == LanguageType.SQL` will always be false under the switch statement `case LanguageType.FLUX`, because there is a `break` ([code](https://github.com/influxdata/ui/pull/6808/files#diff-e221caeec2ec90c4cb59ba36d3dda5fd8990b74997169cf2c79c0172c8571cacR131)) under `case LanguageType.SQL` which separate SQL from Flux. Then there is no need to have `language == LanguageType.SQL` and `sqlAsFlux(text, selection.bucket)`